webpack.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
         filename: '[name].js',
         path: path.resolve(__dirname, 'dist'),
         library: '[name]',
-        libraryTarget: 'umd',
+        libraryTarget: 'amd',
```

Comment:
I think @thornbill said this was technically fine, but I'm not sure which is more desirable here.